include/foonathan/memory/memory_pool.hpp

```diff
@@ -55,8 +55,9 @@ namespace foonathan
             using allocator_type = make_block_allocator_t<BlockOrRawAllocator>;
             using pool_type      = PoolType;
 
+            // actual minimum node size, including any debug fence additions.
             static constexpr std::size_t min_node_size =
-                FOONATHAN_IMPL_DEFINED(free_list::min_element_size);
+                FOONATHAN_IMPL_DEFINED(free_list::actual_node_size(free_list::min_element_size));
```

Comment:
This is one part I'm not 100% sure of. Is `min_node_size` meant to be the minimum amount that can be *requested* (in which case I should not change it), or is it meant to be the minimum you'll actually get, as it is in this update.